package.json

```diff
@@ -28,6 +28,9 @@
     "type": "git",
     "url": "https://github.com/iobroker-community-adapters/ioBroker.shelly"
   },
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
why this requirement? did any dependency require that?

Reply:
We can have it ... but should add it to changelog ... and we do a minor anyway ... I would consider this as enough here